### PR TITLE
New versions of how-to-guides for computing at scale

### PR DESCRIPTION
**Pull request by MMenchero**
## Description:
New versions of the Dask and Ray how-to guides that incorporate the new NixtlaClient class.

## Changes:
- Updated version of issue #175  (this version was too old).
- Tutorials have repetitive information, so they will be consolidated into a single how-to-guide for every distributed framework (Spark, Dask, and Ray) - this is WIP

### CODE REVIEW

File: nbs/docs/how-to-guides/2_distributed_fcst_dask.ipynb [hunk 1]
```diff
@@ -0,0 +1,921 @@
+{
```
Comment (mergenthaler) [inline nbs/docs/how-to-guides/2_distributed_fcst_dask.ipynb:1]: <p>I think there is a word missing here and in the other titles. <code>How to on X</code> does not seem correct.   Maybe <code>How to use Dask: Forecasting</code> or <code>How to Forecast with Dask </code> . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: nbs/docs/how-to-guides/0_computing_at_scale.ipynb [hunk 2]
```diff
@@ -0,0 +1,96 @@
+{
```
Comment (cchallu) [inline nbs/docs/how-to-guides/0_computing_at_scale.ipynb:1]: <p>We should mention that the paralellization is done along different time series in a dataset, so having many time series is also a requirement.</p><p> </p><p>For example: Consists of millions of observations over multiple time series.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (MMenchero) [reply nbs/docs/how-to-guides/0_computing_at_scale.ipynb:1]: Thanks, added a note specifying the need for multiple time series, each with its own unique id.

**Comment by MMenchero** [comment]
This is ready for review @AzulGarza @mergenthaler

**Review by mergenthaler — APPROVED** [review]

**Pull request merged**